### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by sameep-s**
Added Filter Functionality. 

- Sort By price LOW_TO_HIGH & HIGH_TO_LOW.
- filter by brand.
- filter by size.
- filter by range price.
- Clear all Filter

Screenshot (118)

Screenshot (119)

### CODE REVIEW

File: src/pages/ProductList.jsx [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
 import Navbar from './Navbar';
 import Filters from './Filters';
 import CardProduct from './CardProduct';
-import '../pages-css/main.css';
-
+import { useFilter } from '../util_Contexts/filter-context';
+import { getSortedProducts } from '../util_fucntions/getSortedProducts';
+import { dataFilterBrands } from '../util_fucntions/dataFilterBrands';
+import { getFilteredDataSize } from '../util_fucntions/getFilteredDataSize';
+import { getFilteredDataPrice } from '../util_fucntions/getFilteredDataPrice';
```
Comment (arjunbharti) [inline src/pages/ProductList.jsx:12]: you can use an `index.js` file to import all these named exports and import them all using a single file here

**Review by arjunbharti — APPROVED** [review]
Just added a minor suggestion. Have a look
Few things I liked in your code:

- Specific in functions naming style
- Good file structuring
- Well written logic

**Review by cjinc121 — APPROVED** [review]
looks good

File: src/pages/ProductList.jsx (hunk 1 above)
Comment (sameep-s) [reply src/pages/ProductList.jsx:12]: sure i will.

**Pull request merged**